db/migrate/20240709185643_merge_organization_profiles_to_organizations.rb

```diff
@@ -0,0 +1,36 @@
+class MergeOrganizationProfilesToOrganizations < ActiveRecord::Migration[7.1]
+  def up
+    add_column :organizations, :email, :string
+    add_column :organizations, :phone_number, :string
+    add_column :organizations, :donation_url, :text
+    add_column :organizations, :facebook_url, :text
+    add_column :organizations, :instagram_url, :text
+    add_reference :organizations, :location, foreign_key: true
+
+    # Migrate existing data
```

Comment:
I don't think we need these comments. You've written this code self-explanatory 🙂